tests/integrations/raftstore/mod.rs

```diff
@@ -18,6 +17,7 @@ mod test_region_heartbeat;
 mod test_region_info_accessor;
 mod test_replica_read;
 mod test_replication_mode;
+mod test_restart;
```

Comment:
Because these tests are used for testing the behavior of handing commit index after restarting and there is no individual `early apply` feature.

Reply:
The reason why we need to handle commit index is because we apply committed entries before persisting commit index. That's what "early apply" means. If the cases are to check restarting, then there are other things need to be checked. For example, a restarted node should not lose updates, should be able to rejoin the group, etc.